packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -282,8 +282,6 @@ class PerformanceMonitoring {
       return (
         span.duration() > 0 &&
         span._start >= transaction._start &&
-        span._end > transaction._start &&
-        span._start < transaction._end &&
```

Comment:
But even if he adds it, `span._end` will always be less than `transaction.end` if `span._start` is less than the `transaction.end` because if that check does not hold true span.duration will never be more than 0 and we wont reach this expression.

Reply:
Yeah, that makes sense 👍